packages/nc-gui-v2/components/smartsheet/sidebar/MenuTop.vue

```diff
@@ -236,11 +236,7 @@ function onDeleted() {
   }
 
   .active {
-    @apply bg-blue-500/15;
-
-    .nc-icon {
-      @apply !text-pink-500;
-    }
+    @apply bg-primary/20 text-primary font-weight-medium;
```

Comment:
font-weight-medium is probably vuetify as well, is it supposed to be font-semibold?

Reply:
oh it's `font-medium`